cmake/SetupCompilerOptions.cmake

```diff
@@ -111,6 +111,20 @@ if(BLT_DEFINES)
     message(STATUS "Added \"${BLT_DEFINES}\" to definitions")
 endif()
 
+if(COMPILER_FAMILY_IS_MSVC)
+    add_definitions(-D_USE_MATH_DEFINES)
```

Comment:
Yes. 

So, you're suggesting that this should go into the ``config.hpp`` of user projects?